### DISCUSSION TITLE
help can't find sigabert error

### DISCUSSION THREAD

**Discussion by sendhelpp** [sst-inc/help#31] [Q&A - iOS] [answered]
please help I can't find the marked out yellow thing to fix the sigabert error
[contact list v3 5.zip](https://github.com/sst-inc/help/files/6992106/contact.list.v3.5.zip)

**Comment by Joeispro99** [comment, marked as answer]
Ok I found the errors,
Firstly, you set the custom class of your right bar button item to rightBarButton
Secondly, for some reason I think maybe you deleted and recreated your viewcontroller and because its the same name, it was conflicting so I changed the name of the view controller to detail view controller
[contact list v3.zip](https://github.com/sst-inc/help/files/7037031/contact.list.v3.zip)


Should you have any other probelms you can send here again :)

**Reply by Ethan-Chew** [reply]
Please mark this answer as 'answer' if it works for you. Thanks!

**Reply by sendhelpp** [reply]
it works thank
